src/js/components/CheckBox/stories/NoLabel.js

```diff
@@ -13,7 +13,9 @@ const SimpleCheckBox = ({ checked: checkedProp, ...rest }) => {
   );
 };
 
-export const NoLabel = () => <SimpleCheckBox />;
+export const NoLabel = () => (
+  <SimpleCheckBox a11yTitle="No label, but an aria-label" />
```

Comment:
```suggestion
  <SimpleCheckBox a11yTitle="Checkbox without a label" />
```
Maybe something like this gives a screenreader user a little better description of what is displayed

Reply:
Make sense!